REDCapInstrumentAndVariableSQLRenamer.php

```diff
@@ -43,7 +43,7 @@ public function printVariableList($pid): string
                 $aux = $row['form_name'];
                 $list_html .= "<div class='group-header'>" . REDCap::getInstrumentNames($row['form_name']) . "</div>";
             }
-            if ($row['field_name'] != $row['form_name'] . "_complete" && $row['field_name'] != "record_id") {
+            if ($row['field_name'] != $row['form_name'] . "_complete" && $row['field_order'] != "1") {
```

Comment:
@knil-maloon is this the best way to check for the primary key? there isn't another identifier on the table? we just have to see if it is the first one?

Reply:
okay awesome, thank you for doing that.